extensions/adobe/experience/marketo/marketo-person-contact.schema.json

```diff
@@ -0,0 +1,44 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/marketo/marketo-person-contact",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Marketo Contact Schema",
+  "type": "object",
+  "description": "Marketo Contact schema for use with B2B Accounts and Opportunities. Includes other Profile mixins like person/work details, opt in/out etc.",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/profile"],
+  "meta:extends": [
+    "https://ns.adobe.com/xdm/context/profile-contact-crm",
+    "https://ns.adobe.com/experience/marketo/marketo-person"
+  ],
+  "definitions": {
+    "marketo-person-contact": {
```

Comment:
Why do we need to maintain this property in a separate mixin? Merge with marketo-person schema.